db/seeds.rb

```diff
@@ -7,13 +7,13 @@
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
 foodtrucks = [
-    ["Chicken & Rice Guys", "Food truck that specializes in
-      chicken and lamb shawarma.", 4, "Harvard Square"],
-    ["Banh Mi", "Food truck that specializes
-      in Asian fusion-type foods.", 4, "South Station"],
-    ["Mei Mei", "Food truck that serves creative Chinese-American
-      cuisine made
-    from locally sourced and sustainable ingredients.", 4, "Fenway"]
+                ["chicken & rice guys", "Food truck that specializes in
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.